projects/js-packages/publicize-components/src/components/social-previews/use-post-data.js

```diff
@@ -51,38 +51,35 @@ export function usePostData() {
 
 			const media = [];
 
-			// Attach media only if "Share as a social post" option is enabled.
-			if ( shouldUploadAttachedMedia ) {
```

Comment:
Instead of removing this condition altogether, should we instead use `if (media.length)` here?

Reply:
I think media is always empty here because it's created above. I think we do not need the parent if, and just put Sig or attached media into the array if applicable